rskj-core/src/main/java/co/rsk/rpc/modules/trace/TraceModule.java

```diff
@@ -20,8 +20,15 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 
+import java.util.List;
+
 public interface TraceModule {
     JsonNode traceTransaction(String transactionHash) throws Exception;
 
     JsonNode traceBlock(String blockHash) throws Exception;
+
+    JsonNode traceFilter(TraceFilterRequest request) throws Exception;
+
+    JsonNode traceGet(String transactionHash, List<String> positions) throws Exception;
```

Comment:
only `traceGet` is needed? `traceFilter` is part of another PR

Reply:
Yes that is true, we need merge `trace_filter` PR first